contributors/py-helix-admin/helix/statemodeldefs.py

```diff
@@ -0,0 +1,39 @@
+from ordereddict import OrderedDict
+
+# These essentially come from the java classes defined here. It is cheesey and should probably come from a configuration file.
+# https://github.com/linkedin/helix/blob/master/helix-core/src/main/java/com/linkedin/helix/tools/StateModelConfigGenerator.java
+
```

Comment:
Any reason for implementing it like this rather than in a json file or even as a string constant?